mr_popular/mrpopular.py

```diff
@@ -40,22 +40,22 @@ def connect_to_endpoint(url, headers):
 
 
 def handler(event=None, context=None):
-    bearer_token = os.environ.get("BEARER_TOKEN")
+    bearer_token = boto3.client('ssm').get_parameter(Name='TWITTER_BEARER_TOKEN', WithDecryption=True)['Parameter']['Value']
     twooters = os.environ.get("TWITTER_ACCOUNTS")
     url = create_url(twooters)
     headers = create_headers(bearer_token)
     json_response = connect_to_endpoint(url, headers)
     with table.batch_writer() as batch:
         for item in json_response['data']:
             batch.put_item(
-                Item={ 
-                    'EpochTime': int(time.time()),
+                Item={
                     'twitter_handle': item['username'],
+                    'EpochTime': int(time.time()),
```

Comment:
Does the ordering of the attributes to batch_writer matter at all? I think not. I am very intelligent.

Reply:
It does not.  I consider myself OCD that I wanted to change them anyways.